python/pyspark/pandas/tests/data_type_ops/test_categorical_ops.py

```diff
@@ -44,6 +44,26 @@ def other_pser(self):
     def other_psser(self):
         return ps.from_pandas(self.other_pser)
 
+    @property
+    def ordered_pser(self):
+        return pd.Series([1, 2, 3]).astype(CategoricalDtype([3, 2, 1], ordered=True))
+
+    @property
+    def disordered_psser(self):
```

Comment:
nit: Sounds like this should be `unordered_psser`?

Reply:
btw, shall we put this below `ordered_psser`?

Reply:
Good point! Adjusted.